openadapt/start.py

```diff
@@ -24,7 +25,7 @@ def main() -> None:
         subprocess.run(["git", "pull", "-q"])
         logger.info("Updated the OpenAdapt App")
 
-    run_app()  # start gui
+    start()  # start gui
```

Comment:
@shashank40 thank you for putting this together!

@KIRA009 do you know whether we are using this file `openadapt/start.py` anywhere? Git blame shows last update was about a year go

Reply:
@abrichr @KIRA009 as my app was not working after download, i used this script to spin up the app.

`python -m openadapter.start` runs a client for us and that is what i used this for

Reply:
@shashank40 please feel free to move this file to `openadapt/deprecated`, and remove any mention of it in any documentation that you encountered 🙏

Reply:
https://github.com/OpenAdaptAI/OpenAdapt/issues/777

Reply:
@KIRA009 
I meant the compiled app as i commented here.
https://github.com/OpenAdaptAI/OpenAdapt/pull/771#discussion_r1644517094